docs/fleet/_index.md

```diff
@@ -0,0 +1,98 @@
+---
+title: "Fleet Management"
+linkTitle: "Fleet Management"
+weight: 18
+type: "docs"
+description: "Configure, control, debug, and manage your robots from the cloud at app.viam.com."
+no_list: true
+tags: ["fleet management", "cloud", "app"]
+aliases:
+    - /manage/fleet-management
+    - /manage/app-usage
+    - /manage/fleet
+---
+
+On the robot page for each member of your fleet, you have the following tabs:
+
+- [Setup](#setup)
+- [Config](#config)
+- [Logs](#logs)
+- [History](#history)
+- [Code Sample](#code-sample)
+- [Control](#control)
+- [Security](#security)
+
+Soon you will also be able to [deploy code and machine learning models](#package-deployment) to robots.
```

Comment:
I initially thought this meant that after reading this page I'd know how to deploy code etc.

Maybe this shouldn't be here at all if it doesn't exist yet? And certainly not as a highlighted feature (edit: though I now see we're already doing this on the current version anyway)